po/ru.po

```diff
@@ -2151,20 +2152,20 @@ msgstr "Архитектура, которая станет текущей дл
 
 #: app/flatpak-builtins-make-current.c:58
 msgid "APP BRANCH - Make branch of application current"
-msgstr "ПРИЛОЖЕНИЕ ВЕТКА - Сделать ветку текущей для приложения"
+msgstr "ВЕТВЬ ПРИЛОЖЕНИЯ - Сделать ветвь приложения текущей"
```

Comment:
`APP` and `BRANCH` here are placeholders for cli args, and they must be in this order.